### ISSUE TITLE
Feature Request: Add functionality to check S3 bucket logging

### ISSUE THREAD

**Issue by pjuanda** [gruntwork-io/terratest#388]
In my use case, I'll need to check whether a logging is enabled / disabled for an S3 bucket.  I'll add these functions to support this case:
```
GetS3BucketLogging(t *testing.T) *s3.LoggingEnabled
GetS3BucketLoggingE(t *testing.T, region string, bucket string) (*s3.LoggingEnabled, error)
AssertS3LoggingEnabled(t *testing.T, region string, bucket string)
```

**Comment by yorinasub17** [comment]
Thanks for the suggestion! While we do have a lot of functions that wrap the SDK into a better interface for other services, we realized that if we wrapped every function of the SDK, it would be redundant and not very maintainable. So lately we have been taking a step back and trying to focus on exposing functions or routines that we find are especially hard or unintuitive to do directly with the SDK.

Looking at the implementation referenced, I can see that these three functions are fairly thin wrappers around the SDK, which suggest that it might be better to directly use the SDK for these particular checks.

With that said, if you still find these wrappers useful, it would be helpful to know what makes the SDK calls unintuitive to you, and the advantages of these terratest wrappers that has helped you overcome those challenges.
